metrics/timescaledb/db.py

```diff
@@ -68,30 +69,29 @@ def _batch_size(table):
     return max_params // len(table.columns)
 
 
-def _drop_table(table, batch_size):
-    with _get_engine().begin() as connection:
-        log.debug("Removing table: %s", table.name)
+def _drop_table(connection, table, batch_size, cascade=False):
```

Comment:
Did `cascade` creep in when you were refactoring?

Reply:
Ugh. Well spotted. Thanks.